core/serialisation/src/test/java/uk/gov/gchq/gaffer/serialisation/implementation/ordered/OrderedDateSerialiserTest.java

```diff
@@ -17,14 +17,17 @@
 package uk.gov.gchq.gaffer.serialisation.implementation.ordered;
 
 import org.junit.Test;
+import uk.gov.gchq.gaffer.commonutil.pair.Pair;
 import uk.gov.gchq.gaffer.exception.SerialisationException;
+import uk.gov.gchq.gaffer.serialisation.Serialiser;
+import uk.gov.gchq.gaffer.serialisation.ToBytesSerialisationTest;
 import java.util.Date;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
-public class OrderedDateSerialiserTest {
+public class OrderedDateSerialiserTest extends ToBytesSerialisationTest<Date> {
 
     private static final OrderedDateSerialiser SERIALISER = new OrderedDateSerialiser();
```

Comment:
This field is hiding a duplicate field in the super class test.
Delete this field and rename all references to lowercase "serialiser"
Repeat for all other tests in this pull request.